package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "start": "npm run build -- -w",
-    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
```

Comment:
=> `webpack-defaults`, but `options.json` in the root is also ok imho,  since it is metadata, still needs to be added to `pkg.files` then, so really no preference :)